Blog/views.py

```diff
@@ -146,6 +147,12 @@ class PostCreate(LoginRequiredMixin, generic.CreateView):
     def form_valid(self, form):
         form.instance.user = self.request.user
         form.instance.created = datetime.datetime.now()
+        same_titles_count = Post.objects.filter(title=form.instance.title).count()
```

Comment:
The application crashes under the following circumstances:

- Create a blog post having the title `Test`, this works and I can access the post under http://localhost:8000/en/blog/post/test
- Create another blog post having the title `TEST` (all upper case)

Then you get an error:

```
MultipleObjectsReturned at /en/blog/post/test
get() returned more than one Post -- it returned 2!
```

Hints: 
- Before creating the blog post, you could query the database for blog posts having the same slug, not the same title, because `Test` and `TEST` both result in the same slug `test`. You could create form validator for this.

Reply:
Thanks for the hint - wouldn't it be easier to just filter case insensitively?
I. e. `same_titles_count = Post.objects.filter(title__iexact=form.instance.title).count()` at line 150

The bug is, that the titles are not counted correctly, with this solution they would.

Reply:
This would not solve the case, where a blog post titled `Foo-Bar` clashes with another one titled `Foo Bar` because both titles result in `foo-bar`.

So I would slugify the title, then query the database for other blog posts having this slug and append the suffix accordingly.

Reply:
By the way, you can ignore my hint about the form validation. Because we don't want to the user to change the title of the blog post, just because the title would result in a slug already existing in the database 🤔